web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -837,6 +844,7 @@
       xhr.open("GET", url, true);
       xhr.timeout = settings.cometGetTimeout;
       xhr.setRequestHeader("Accept", "text/javascript, application/javascript, application/ecmascript, application/x-ecmascript, */*; q=0.01");
+      xhr.setRequestHeader("X-Requested-With", "XMLHttpRequest"); // So Req.ajax_? will return true
```

Comment:
IMHO, we can leave off the comment about `Req.ajax_?` in both places it exists. Can keep it if you feel strongly otherwise, though.